platforms/access-bridge/src/app-config.ts

```diff
@@ -6,6 +6,10 @@ function requireEnvVar(env: string | undefined, name: string): string {
   return env;
 }
 
+// Secrets resonsible for authenticating requests
+export const API_SECRET = requireEnvVar(process.env.APP_API_SECRET, 'APP_API_SECRET');
```

Comment:
We've had a couple conversations about vite and env vars but I don't know if we've talked about these... are these intended to be injected at build time or runtime? Runtime is probably better, since we can then re-use the same build for different customers.

Reply:
I had them in run time using dotEnv, but one of your suggestion was to use vite to inject them, so now they are injected at build time.
Btw, how can we reuse single deployment to different clients? They will still need to deploy it on their side.